Readme.md

```diff
@@ -6,20 +6,18 @@
+Controls usually have the same settings for different pages of a large web application, and we recommend that you specify the component settings in markup. You can also override the **SetParametersAsyncCore** (for Grid) or **SetParametersAsync** (for other components) method to specify the default settings for Blazor components in your application. However, we do not recommend that you use this technique because it is uncommon in Blazor. Use it at your own risk.
 
 ![Grid with predefined settings and default](images/result.png)
 
-Create an inheritor from our Blazor control that overrides the **SetParametersAsync** method. In this method, set the default values for the public properties before you call the base method. You can initialize properties with custom default values in markup because the base method is called after your custom settings. We do not recommend that you override and change the values of protected properties.
+Inherit from our Blazor control to override the **SetParametersAsync** or **SetParametersAsyncCore** method. In one of these methods, set the default values for the public properties before you call the base **SetParametersXXX** method. In markup, you can initialize properties with modified default values because the base method is called after custom settings are applied. We do not recommend that you override and change the values of protected properties.
 
-In this example, the [MyDataGrid](./CS/DxBlazorComponentsDefaultSettings/Components/MyDataGrid.cs) component contains the inherited **DxDataGrid** component whose **PageSize**, **ShowFilterRow**, and **ShowGroupPanel** properties contain new default values. You can compare the **MyDataGrid** component with the default **DxDataGrid** component on the [Index](./CS/DxBlazorComponentsDefaultSettings/Pages/Index.razor) page to ensure they have the same settings in markup.
+In this example, the [MyGrid](./CS/DxBlazorComponentsDefaultSettings/Components/MyGrid.cs) component contains the inherited **DxGrid** component whose **PageSize**, **ShowFilterRow**, **AllowSort**, and **ShowGroupPanel** properties contain new default values. You can compare the **MyGrid** component to the original **DxGrid** component on the [Index](./CS/DxBlazorComponentsDefaultSettings/Pages/Index.razor) page to ensure they have the same settings in markup.
 
-The Data Grid was moved to maintenance support mode. No new features/capabilities will be added to this component. We recommend that you [migrate](https://docs.devexpress.com/Blazor/403162/grid/migrate-from-data-grid-to-grid) to the [Grid](https://docs.devexpress.com/Blazor/403143/grid) component. 
-
-You cannot use this approach for **DxChartXXXSeries** components and specify the **ValueField**, **ArgumentField**, **NameField**, and **PaneField** properties.
+You cannot use this technique for **DxChartXXXSeries** components and specify the **ValueField**, **ArgumentField**, **NameField**, and **PaneField** properties.

```

Comment:
```suggestion
You cannot use this technique for **DxChartXXXSeries** components and specify the **ValueField**, **ArgumentField**, **NameField**, and **PaneField** properties.

```
You haven't mentioned **DxChartXXXSeries** components before. How does this relate to this help doc? This last paragraph doesn't seem to flow with the one before. Is there something missing here that should be said. Let me know.

Reply:
As the example title says, the approach shown here can be applied to all our Blazor components. The internal implementation of the Chart series requires specifying **ValueField**, **ArgumentField**, **NameField**, and **PaneField** directly in markup to define series data types and values correctly.

Reply:
ok